bindings/python/src/datetime.cpp

```diff
@@ -55,6 +55,29 @@ struct time_duration_to_python
     }
 };
 
+struct time_point_to_python
+{
+    static PyObject* convert(lt::time_point pt)
+    {
+        using boost::chrono::system_clock;
+        using boost::chrono::duration_cast;
+        time_t const tm = system_clock::to_time_t(system_clock::now()
+            + duration_cast<system_clock::duration>(pt - lt::clock_type::now()));
```

Comment:
I added this to code:
```
std::cout << tm << "\n";
std::cout << now << "\n";
std::cout << pt << "\n";
```
resulting in:
```
10709678762
100238916604617 nanoseconds since boot
-9223372036854775808 nanoseconds since boot
```
So it's a rather large negative number?